modules/utxo-lib/test/integration_local_rpc/generate/fixtures.ts

```diff
@@ -25,8 +25,11 @@ export function getProtocolVersions(network: Network): number[] {
 }
 
 export function getFixtureDir(protocol: Protocol): string {
-  const networkName = getNetworkName(protocol.network);
+  let networkName = getNetworkName(protocol.network);
   assert(networkName);
+  if (networkName === 'bitcoinBitGoSignet') {
```

Comment:
Wouldn't a coin name of `tbtcbg` make more sense if the network is written like that?